doc/classes/RigidBody3D.xml

```diff
@@ -158,7 +158,7 @@
 			See [method add_constant_torque].
 		</member>
 		<member name="contact_monitor" type="bool" setter="set_contact_monitor" getter="is_contact_monitor_enabled" default="false">
-			If [code]true[/code], the RigidBody3D will emit signals when it collides with another RigidBody3D.
+			If [code]true[/code], the RigidBody3D will emit signals when it collides with another body.
```

Comment:
Or maybe this would be clearer?

```suggestion
			If [code]true[/code], the RigidBody3D will emit signals when it collides with another [PhysicsBody3D].
```

Reply:
I considered it, but since it also reports TileMaps, I thought it was better to be generic.